deno/payloads/v10/auditLog.ts

```diff
@@ -826,6 +830,27 @@ export type APIAuditLogChangeKeyExemptRoles = AuditLogChangeData<'exempt_roles',
  */
 export type APIAuditLogChangeKeyExemptChannels = AuditLogChangeData<'exempt_channels', Snowflake[]>;
 
+/**
+  * Returned when a guild forum's available tags gets changed
+ */
+export type APIAuditLogChangeKeyAvailableTags = AuditLogChangeData<'available_tags', APIGuildForumTag[]>
```

Comment:
You need to add these in the payloads/v10, payloads/v9, then run the scripts and it'll update the deno files
Also linting